Gopkg.toml

```diff
@@ -17,13 +16,18 @@
 #   source = "github.com/myfork/project2"
 #
 # [[override]]
-#  name = "github.com/x/y"
-#  version = "2.4.0"
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
 
 
 [[constraint]]
   name = "code.cloudfoundry.org/lager"
-  branch = "master"
```

Comment:
We shouldn't change the dependency versions

Reply:
This might have changed after I did a `dep ensure` to run the test. Will revert this one.